### DISCUSSION TITLE
How to simulate a multi-cloud environment?

### DISCUSSION THREAD

**Discussion by LeeBoonKong** [manoelcampos/cloudsim-plus#359] [Q&A] [answered]
I would like to ask how I can simulate a Multi-Cloud environment in Cloudsimplus? 
Is there an example for this?

**Comment by manoelcampos** [comment, marked as answer]
Hello @LeeBoonKong 

Despite there isn't the concept of multi-cloud on CloudSim Plus, if you consider you have 4 datacenters where:

- one cloud provider has 2 datacenters,
- another provider has other 2 datacenters

we could say we have a multi-cloud environment.

However, VM migration between different datacenters was not supported in CloudSim or CloudSim Plus.
That was a feature that have been waiting a long time to be implemented.
I fixed it in #361 and introduced a completely new example. Check the link there.
